toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -702,6 +708,14 @@ void FarmServer::queryHwInfo(HwInfo &hwInfo)
 	len = sizeof(physMemSize);
 	sysctl(mib, 2, &physMemSize, &len, NULL, 0);
 
+#elif defined(LINUX)
+	physMemSize = (TINT64)sysconf(_SC_PHYS_PAGES) * (TINT64)sysconf(_SC_PAGE_SIZE);
```

Comment:
The purpose of this check is to find the physical memory on the system, not virtual memory.
So not sure why you would even consider virtual memory.

----

Changes to actual logic for testing memory may be fine, but outside the scope of porting code.